api/src/main/resources/default.mapping

```diff
@@ -25,6 +21,7 @@ javax/security/auth/message/=jakarta/security/auth/message/
 javax/security/enterprise/=jakarta/security/enterprise/
 javax/security/jacc/=jakarta/security/jacc/
 javax/servlet/=jakarta/servlet/
+javax/transaction/H=jakarta/transaction/H
 javax/transaction/I=jakarta/transaction/I
 javax/transaction/N=jakarta/transaction/N
 javax/transaction/R=jakarta/transaction/R
```

Comment:
javax.xml.registry is removed in EE 9, so we should think about adding an option for handling that, if we are transforming an EE 8 application to EE 9 and they use javax.xml.registry, that shouldn't be changed to jakarta.xml.registry.

I did raise an issue for that on [the Eclipse Transformer](https://github.com/tbitonti/jakartaee-prototype/issues/79).

Reply:
Fixed